erc/ERCS/erc-4337.md

```diff
@@ -115,23 +134,33 @@ The account:
   * `validAfter` is 6-byte timestamp. The UserOp is valid only after this time.
   
 An account that works with aggregated signature, should return its signature aggregator address in the "sigAuthorizer" return value of validateUserOp.
-It MAY ignore the signature field
+It MAY ignore the signature field.
+
+The account MAY implement the interface `IAccountExecute`
+
+```solidity
+interface IAccountExecute {
+  function executeUserOp(PackedUserOperation calldata userOp, bytes32 userOpHash) external;
+}
+```
+
+This method will be called by the entryPoint with the current UserOperation.
```

Comment:
I suggest making the `executeUO` a paragraph.
Also, this sentence doesn't fully describe the behavior - add "will be called instead of just calling with the `calldata`" and all that, maybe?